db/migrate/20240501000631_create_decision_review_evidence_validation.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDecisionReviewEvidenceValidation < ActiveRecord::Migration[7.1]
+  def change
+    create_table :decision_review_evidence_validations do |t|
+      t.uuid :decision_review_guid
```

Comment:
just for clarity since there are a gazillion UUIDs 😂 which ID is this representing?

Reply:
Sorry about that - I can change it to `decision_review_evidence_attachment_guid`

Reply:
Is it worth tossing an index on this field as well? How many are we thinking will be in here?

Reply:
I can add an index - hard to say what percentage of the files are actually encrypted (on average I am seeing 300-500 evidence uploads a day).

Reply:
Thanks @dfong-adh !